Xamarin.Forms.Platform.iOS/Renderers/TabbedRenderer.cs

```diff
@@ -363,6 +372,18 @@ void UpdateBarTextColor()
 			TabBar.TintColor = isDefaultColor ? _defaultBarTextColor : barTextColor.ToUIColor();
 		}
 
+		void UpdateTintColor()
+		{
+			if (Tabbed == null || TabBar == null || TabBar.Items == null)
+				return;
+
+			if (SelectedTintColor != Color.Default)
```

Comment:
what if you want to return to the default color?  you will need to save the default color of the platform and use it when the Color is set to Default. and had change before.